template/default.tmpl

```diff
@@ -65,8 +65,7 @@ AlertmanagerUrl:
 {{ define "wechat.default.to_party" }}{{ end }}
 {{ define "wechat.default.to_tag" }}{{ end }}
 {{ define "wechat.default.agent_id" }}{{ end }}
-
-
+{{ define "wechat.default.chat_id" }}text{{ end }}
```

Comment:
You can remove the `text` part. You should also add something for the other new fields.